lib/base_interceptor.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+require 'gruf'
+
+module GrpcRest
+    # This is a monkey-patch that fixes an issue we were having with using the fail! method in
+    # interceptors where an active call was not instantiated yet.
+    # Basically, we overloaded this function: https://github.com/bigcommerce/gruf/blob/main/lib/gruf/errors/helpers.rb#L34
```

Comment:
I'd change this comment to:

```
Fixes an issue with the fail! method since the active call is not instantiated yet.
Overloads https://github.com/bigcommerce/gruf/blob/main/lib/gruf/errors/helpers.rb#L34
```

Reply:
Makes sense, less filler words. 👍